scripts/releaseV2/phase1-bump-all-packages.mjs

```diff
@@ -53,8 +57,17 @@ const rootFolder = resolve(dirname(fileURLToPath(import.meta.url)), '..', '..');
   }
   const parsedCommits = parseCommits(commits, convention.parserOpts);
   const bumpInfo = convention.recommendedBumpOpts.whatBump(parsedCommits);
-  const currentVersion = getCurrentVersion(PATH);
-  const newVersion = getNextVersion(currentVersion, bumpInfo);
+  let currentVersion = getCurrentVersion(PATH);
+  if (isPrerelease) {
+    const currentBetaNpmVersion = await describeNpmTag(
+      packageJson.name,
+      'beta'
+    );
+    currentVersion = gt(currentBetaNpmVersion, currentVersion)
```

Comment:
when would the `currentVersion` would be greater than `currentBetaNpmVersion`?

Reply:
Let's say I start with 1.0.0 for both the latest and beta.
I do a commit with a feature on master. The 1.1.0-beta1 is created and beta point to it. 1.1.0-beta1>1.0.0, thus beta>latest. So far so good.
I now do a release. The 1.1.0 is created and latest point to it. 1.1.0>1.1.0-beta1, thus latest>beta.